packages/preact-router/src/utils.ts

```
@@ -0,0 +1,120 @@
+import {
+  useRef,
+  useEffect,
```

ESLint

[error] Member 'useEffect' of the import declaration should be sorted alphabetically.

(sort-imports)